src/LibraryManager.Build/Microsoft.Web.LibraryManager.Build.csproj

```diff
@@ -13,7 +13,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Framework" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" />
-    <PackageReference Include="System.Runtime.Loader" Condition="'$(TargetFramework)' == 'netstandard2.0'"/>
```

Comment:
This is to resolve the new use of `AssemblyLoadContext` in `src/LibraryManager.Build/Contracts/Dependencies.cs`